samples/subsys/usb/webusb/README.rst

```diff
@@ -1,21 +1,20 @@
-Title: WebUSB Enabled Custom Driver and A Sample Application
+.. _webusb-sample:
 
-This project includes:
+=========================
```

Comment:
Take a look at http://docs.zephyrproject.org/contribute/doc-guidelines.html for our reST guidelines.

The document title just uses an underline (no overline) with ``#``s and the heading hierarchy after that is ``*``, ``=``, and ``-``.

Change the document title here to be just:

    WebUSB sample application
    #########################